lake-manifest.json

```diff
@@ -40,5 +32,13 @@
     "opts": {},
     "name": "proofwidgets",
     "inputRev?": "v0.0.21",
+    "inherited": false}},
+  {"git":
+   {"url": "https://github.com/leanprover/std4",
+    "subDir?": null,
+    "rev": "bd60d932e2c786c7347e57576598568b2816e316",
+    "opts": {},
+    "name": "std",
+    "inputRev?": "main",
```

Comment:
Do you need this re-ordering of lake-manifest in this PR?

Reply:
Of course not, but lake *loves* to shuffle this file.